server/command.go

```diff
@@ -273,7 +253,7 @@ func (p *Plugin) handleSubscribe(_ *plugin.Context, args *model.CommandArgs, par
 			return fmt.Sprintf("%s Though there was an error creating the public post: %s", msg, appErr.Error())
 		}
 
-		return msg
+		return ""
```

Comment:
Why not return `msg`?

Reply:
@mickmister can get the context here behind this logic https://github.com/mattermost/mattermost-plugin-github/pull/517#issuecomment-989178101 , later on this decided to show only a public post on successful subscription